LATEST

```diff
@@ -1,5 +1,5 @@
 {
-  "daml": "2.3.0-snapshot.20220509.9874.0.0798fe15",
-  "canton": "20220510",
+  "daml": "2.3.0-snapshot.20220614.10077.0.9c5c72a2",
+  "canton": "20220615",
```

Comment:
Does this go here?

Reply:
Yes? I'm not sure I follow the question. This is indeed the place where we put the Canton version.

If you're asking whether the version bump should be part of this PR, strictly speaking, no, it doesn't need to be. I just noticed it'd been a while since we bumped it and thought I might as well do it while I was there.

This will update the live https://docs.daml.com/2.3.0/ once merged.